build-system/server.js

```diff
@@ -19,7 +19,7 @@
  * @fileoverview Creates an http server to handle static
  * files and list directories for use with the gulp live server
  */
-const app = require(require.resolve('./app.js'));
+const app = require.resolve('./app.js');
```

Comment:
I think this can be written as 

```
const {app} = require('./app');
```

Reply:
Whoops. My usage of `resolve` was vey wrong as well.

Reply:
I think this is still wrong. Note the curly braces around `{app}`.

Reply:
That wasn't it, but culprit has been identified.